packages/journalforing/src/components/journalpost/JournalpostDetaljer.tsx

```diff
@@ -119,10 +125,55 @@ const JournalpostDetaljer: FunctionComponent<OwnProps> = ({
   }, []);
   const isSubmittable = formMethods.formState.isDirty;
 
+  const reserverOppgaveAction = useCallback(() => {
+    reserverOppgave({
+      oppgaveId: oppgave.id.toString(),
+      versjon: oppgave.versjon,
+      reserverFor: !oppgave.reservertAv ? navAnsatt.brukernavn : '',
+    });
+  }, []);
```

Comment:
Det kan vera dette funkar fint (og at du er klar over korleis dette funkar). Men for sikkerheits skyld... Om oppgave som blir sent inn til JournalpostDetaljer endrar seg så vil ikkje dette funka særleg bra. Sidan dependency-lista til useCallback er tom så vil denne funksjonen kun bli laga første gongen koden blir køyrt. Med andre ord blir den første oppgava sendt med inn alltid brukt når reserverOppgaveAction bli kalla. Om oppgava kan endrast må den leggast til dependency-lista. (Om JournalpostDetaljer alltid blir unmounta/mounta på nytt ved skifta av oppgave så vil det funka)

Reply:
Oppgaven ikke kan endres før man trykker på knappen - men så etter man trykker så får man en oppdatert versjon av oppgaven. Så mulig det er bedre å legge inn oppgaven inn i deps lista. Takk